tests/testthat/_snaps/operators.md

```diff
@@ -0,0 +1,14 @@
+# %*% errors informatively
+
+    Code
+      a %*% b
+    Error <simpleError>
+      incompatible dimensions: "3x4" vs "1x4"
+
+---
+
+    Code
+      a %*% c
+    Error <simpleError>
+      cannot coerce type 'builtin' to vector of type 'character'
```

Comment:
not so informative, but maybe this is a silly test? Or is it an error in the error-message code?

Reply:
I'm actually not sure what is happening here, might need to take a closer look.

Reply:
Passed `stop` `dim` instead of the `msg` !